src/components/VmUserMessages/style.css

```
@@ -1,7 +1,21 @@
 .panel-body {
-    border-bottom: 1px solid #d1d1d1;
     padding: 0;
     background-color: #fff;
+    overflow-y: auto;
+    display: flex;
+    flex-direction: column;
+}
+
+.drawer-pf-notifications-list {
+    border: solid 1px #d1d1d1 !important;
+    padding: 10px 0 !important;
+    margin: 0;
+    overflow-y: auto;
+}
```

```suggestion
.notifications-list {
    overflow-y: auto;
}
```

With the way css is import to the components, we can use shorter class names.

The border, padding and margins aren't necessary.  The extra padding makes the empty state look strange.